app/src/main/java/uk/co/neuralcubes/neuralates/muse/MuseHandler.java

```diff
@@ -0,0 +1,266 @@
+package uk.co.neuralcubes.neuralates.muse;
+
+import android.support.annotation.NonNull;
+
+import com.google.common.eventbus.EventBus;
+import com.interaxon.libmuse.Accelerometer;
+import com.interaxon.libmuse.Battery;
+import com.interaxon.libmuse.Eeg;
+import com.interaxon.libmuse.Muse;
+import com.interaxon.libmuse.MuseArtifactPacket;
+import com.interaxon.libmuse.MuseConnectionListener;
+import com.interaxon.libmuse.MuseConnectionPacket;
+import com.interaxon.libmuse.MuseDataListener;
+import com.interaxon.libmuse.MuseDataPacket;
+import com.interaxon.libmuse.MuseDataPacketType;
+
+import java.io.Closeable;
+
+/**
+ * Created by javi on 10/04/16.
+ */
+
+/**
+ * This class connects to a paired muse and emits a series of events to the event bus to inform
+ * of the changed states
+ *
+ * This events are:
+ *
+ *  <ul>
+ *     <li> com.interaxon.libmuse.ConnectionState </li>
+ *     <li> com.interaxon.libmuse.Muse </li>
+ *     <li> uk.co.neuralcubes.MuseHandler.AccelerometerReading </li>
+ *     <li> uk.co.neuralcubes.MuseHandler.FocusLevel </li>
+ *     <li> uk.co.neuralcubes.MuseHandler.HorseshoeReading </li>
+ *  </ul>
+ *
+ *
+ */
+public class MuseHandler implements Closeable {
+
+
+
+
+    class AccelerometerReading {
+
+        private double x,y,z;
+
+        public AccelerometerReading(double x, double y, double z) {
+            this.z = z;
+            this.y = y;
+            this.x = x;
+        }
+
+
+        double getX(){
+            return x;
+        }
+        double getY(){
+            return y;
+        }
+        double getZ(){
+            return z;
+        }
+    }
+
+    class BatteryReading{
```

Comment:
They encapsulate values so we can send them down the EventBus, otherwise the subscribed observers wouldn't know which class expect. We could refactor the communication to Listener interface instead of an event bus